### PR TITLE
NXP-26851: move from gulp and polymer-build to npm scripts and webpack

### PR DESCRIPTION
**Pull request by nelsonsilva**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,36 +6,32 @@
...
+    "webpack-bundle-analyzer": "^3.0.3",
+    "webpack-cli": "^3.1.2",
+    "webpack-dev-server": "^3.1.10",
+    "webpack-merge": "^4.1.4",
+    "workbox-cli": "3.6.3"
   },
   "scripts": {
-    "build": "gulp build",
-    "start": "gulp serve",
+    "start": "webpack-dev-server --env development",
```
Comment (Gabez0r) [inline package.json]: If you run `start` without running build first, you won't get a  `.tmp` file, and we won't get merged labels either. We can fix it by adding a `prestart` script:
```suggestion
    "prestart": "mkdir -p .tmp && npm run prepare:i18n",
    "start": "webpack-dev-server --env development",
```

File: scripts/merge-messages.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+const DEST = '.tmp/i18n';
+const CWD = process.cwd();
+
+fs.mkdirSync(DEST, {recursive: true});
+
+glob('i18n/messages*.json', (_, files) => files.forEach(file => {
+  let messages = require(`${CWD}/${file}`);
+  let filename = path.basename(file);
+  let p = require(`${CWD}/node_modules/@nuxeo/nuxeo-ui-elements/i18n/${filename}`);
```
Comment (Gabez0r) [inline scripts/merge-messages.js]: Needs to be a path here:
```suggestion
  let p = `${CWD}/node_modules/@nuxeo/nuxeo-ui-elements/i18n/${filename}`;
```

**Comment by Gabez0r** [comment]
👍

**Comment by guirenard** [comment]
:+1:

**Pull request merged**